.changeset/weak-chefs-sniff.md

```diff
@@ -0,0 +1,6 @@
+---
+'@directus/env': patch
+'@directus/api': patch
+---
+
+Deprecated KEY env var, make SECRET optional
```

Comment:
I'd mention right away in the changeset that it is only suitable for test purposes

```suggestion
Deprecated KEY env var, made SECRET optional (for non-productive environments)
```

Reply:
~~productive~~ production 🙂

Reply:
Oh right 🙈😅